src/http-gateways/web-redirects-file.md

```diff
@@ -75,7 +75,11 @@ For example:
 
 This rule will redirect a URL like `/posts/06/15/2022/hello-world` to `/articles/2022/06/15/hello-world`.
 
-### Splat
+Implementation MUST error when the same placeholder name is used more than once in `from`.
```

Comment:
I think I violate this one currently. /:a/:a would I think substitute the latter into to

But that's confusing-at-best. Error makes more sense.

Reply:
I think this is saying that not only that directive becomes a 500 response... but that any request going through the file becomes a 500, right?

Reply:
Yes, a duplicate `:placholder` in `from` is kind of a syntax error, and we want to HTTP 500 when those happen, so website owner fixes the problem, and not relies on unspecified behavior.